types/svg-path-parser/index.d.ts

```diff
@@ -6,49 +6,62 @@
 export function parseSVG(input: string): Command[];
 export function makeAbsolute(commands: Command[]): Command[];
 
-export interface Command {
+export type Command = (
+    MoveToCommand |
+    LineToCommand |
+    HorizontalLineToCommand |
+    VerticalLineToCommand |
+    ClosePathCommand |
+    CurveToCommand |
+    SmoothCurveToCommand |
+    QuadraticCurveToCommand |
+    SmoothQuadraticCurveToCommand |
+    EllipticalArcCommand
+);
+
+export interface BaseCommand {
```

Comment:
I'm not sure you're gaining anything by declaring (and exporting?) a base type.  If you decide to keep it, you may wish to drop the re-declarations of `relative` from the derived types.

Reply:
I wanted to keep the diff quite minimal, and this shape had been exported in the earlier version too. I see no harm in exporting it, in case someone wants to do e.g. `X extends BaseCommand["code"]` for an enum-ish code type...

Reply:
In that case, you may wish to remove the re-declarations of `relative` from the derived types.